docs/examples/spam/pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[build-system]
+requires = [
+    "meson>=0.63.0",
+    "meson-python",
+    "patchelf",
+    "wheel",
```

Comment:
Only `meson-python` should be needed here, as it will request the necessary dependencies itself. Unless the Meson version needs extra bounds.